Adyen/Components/AbstractPersonalInformationComponent/FirstNameFormItemInjector.swift

```diff
@@ -0,0 +1,42 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+internal final class FirstNameFormItemInjector: FormItemInjector, Localizable {
```

Comment:
Both myself and Sonar think this is a duplication :D
We can probably address this in another PR..

Reply:
yep, this is fixed :-).